### PR TITLE
LPD-43551 Add company scoped global CSS sample and functional test

### PR DESCRIPTION
**Pull request by thiago-buarqque**
Waiting https://github.com/liferay-devtools/liferay-portal/pull/315 to be merged

## Goal

Add the sample and the functional test for company scoped Global CSS CXs.

## Approach

I have updated the existing sample to be numbered as the first one, mine will be second

### CODE REVIEW

File: modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java [hunk 1]
```diff
@@ -105,7 +106,17 @@ public boolean isShowBreadcrumb() {
 
 	private Predicate<CET> _getPredicate(String type) {
 		if (Objects.equals(
-				type, ClientExtensionEntryConstants.TYPE_GLOBAL_JS)) {
+				type, ClientExtensionEntryConstants.TYPE_GLOBAL_CSS)) {
+
+			return cet -> {
+				GlobalCSSCET globalJSCET = (GlobalCSSCET)cet;
```
Comment (algobob) [inline modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java:112]: typo in variable name :mag:

**Review by algobob — CHANGES_REQUESTED** [review]
Just one typo :slightly_smiling_face:

File: modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java (hunk 1 above)
Comment (thiago-buarqque) [reply modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java:112]: Ops, forgot to mention to just consider the last three commits @algobob

Comment (algobob) [reply modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java:112]: oh cool

File: modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/service/taglib/ClientExtensionTopHeadDynamicInclude.java [hunk 2]
```diff
@@ -61,13 +68,40 @@ public void include(
...
+					continue;
+				}
+
+				_writeStyleSheet(
+					httpServletRequest, printWriter, globalCSSCET.getURL());
+			}
+		}
+		catch (Exception exception) {
+			_log.error(
+				"Unable to inject global CSS client extensions for company " +
```
Comment (ethib137) [inline modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/service/taglib/ClientExtensionTopHeadDynamicInclude.java:102]: We should just refer to this as "CSS client extension". "global" should be removed.

File: modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java (hunk 1 above)
Comment (ethib137) [reply modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java:112]: @thiago-buarqque why is this included if it's not supposed to be reviewed?

**Review by ethib137 — CHANGES_REQUESTED** [review]
Thanks @thiago-buarqque

**Review by ethib137 — APPROVED** [review]
LGTM

**Comment by thiago-buarqque** [comment]
Sent to https://github.com/liferay-devtools/liferay-portal/pull/325

**Pull request closed without merging**